### PR TITLE
missing translation for "!area overview" command

### PR DESCRIPTION
**Pull request by ReuschelCGN**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: locale/de.json [hunk 1]
```diff
@@ -422,6 +422,7 @@
     "Ordinary":"Standardform",
     "Origin":"Ur",
     "Overcast":"Wolken",
+    "overview":"übersicht",
```
Comment (Fabio1988) [inline locale/de.json:425]: Really lowercased?

Comment (ReuschelCGN) [reply locale/de.json:426]: Dosent matter in commands will you prefer uppercase?

Comment (Fabio1988) [reply locale/de.json]: Oh sure, you are right

File: locale/de.json [hunk 2]
```diff
@@ -390,6 +390,7 @@
     "Mossy Lure Module":"Moos-Lockmodul",
     "Mossy Lure":"Moos Lockmodul",
     "mossy":"moos",
+    "move":"Attacke",
```
Comment (jfberry) [inline locale/de.json:393]: this is used as a parameter for track, does it work with upper case A?

Comment (ReuschelCGN) [reply locale/de.json:393]: yes

Comment (ReuschelCGN) [reply locale/de.json:393]: yes it works

**Comment by ReuschelCGN** [comment]
the commands parameter are not case sensitive, so it works both

**Pull request merged**